packages/net/http/payload.pony

```diff
@@ -243,3 +242,26 @@ class iso Payload
     end
 
     list
+
+  fun string(): String =>
+    var str = proto + status.string() + method + "\n"
```

Comment:
This pattern is covered in the Pony Patterns book as well:

http://patterns.ponylang.org/performance/limiting-string-allocations.html